Documentation/ceph-common-issues.md

```diff
@@ -817,3 +818,70 @@ If you still decide to configure an OSD on LVM, please keep the following in min
 - Avoid incrementing the `count` field of `storageClassDeviceSets` and create a new LV that backs a OSD simultaneously.
 
 You can know whether the above-mentioned tag exists tag with the command: `sudo lvs -o lv_name,lv_tags`. If the `lv_tag` field is empty in an LV corresponding to the OSD lv_tags, this OSD encountered the problem. In this case, please [retire this OSD](ceph-osd-mgmt.md#remove-an-osd) or replace with other new OSD before restarting.
+
+## The Ceph-CSI driver was being unexpectedly removed
```

Comment:
Typically I would say we want documentation for environmental issues and bug fixes like this wouldn't need to be in the documentation. Since this bug existed for so long I'm ok with adding it to the documentation if you find it useful, although it will quickly not be needed after users upgrade.

Reply:
OK, I'll close this ticket. Thank you for your explanation. And @Madhu-1 , thank you for your review!